app/mapping/playback/LeadTrusteeIndExtractor.scala

```diff
@@ -0,0 +1,110 @@
+
+  import PlaybackAddressImplicits._
+
+  override def extract(answers: UserAnswers, data: Option[DisplayTrustLeadTrusteeIndType]): Either[PlaybackExtractionError, Try[UserAnswers]] =
+    {
+      data match {
+        case None => Right(Success(answers))
+        case leadTrustee =>
+
+          val updated = leadTrustee.foldLeft[Try[UserAnswers]](Success(answers)){
+            case (answers, leadTrustee) =>
+
+              answers
+                .flatMap(_.set(IsThisLeadTrusteePage(0), true))
+                .flatMap(_.set(TrusteeIndividualOrBusinessPage(0), IndividualOrBusiness.Individual))
+                .flatMap(_.set(TrusteesNamePage(0), FullName(leadTrustee.name.firstName, leadTrustee.name.middleName, leadTrustee.name.lastName)))
+                .flatMap(_.set(TrusteesDateOfBirthPage(0),
+                  LocalDate.of(leadTrustee.dateOfBirth.getYear, leadTrustee.dateOfBirth.getMonthOfYear, leadTrustee.dateOfBirth.getDayOfMonth)))
+                .flatMap(answers => extractNino(leadTrustee, answers))
+                .flatMap(answers => extractPassportOrIDCard(leadTrustee, answers))
+                .flatMap(answers => extractAddress(leadTrustee, answers))
+                .flatMap(_.set(TelephoneNumberPage(0), leadTrustee.phoneNumber))
+                .flatMap(_.set(EmailPage(0), leadTrustee.email))
+                .flatMap(_.set(TrusteesSafeIdPage(0), leadTrustee.identification.safeId))
+                .flatMap {
+                  _.set(
+                    LeadTrusteeMetaData(0),
+                    MetaData(
+                      lineNo = leadTrustee.lineNo,
+                      bpMatchStatus = leadTrustee.bpMatchStatus,
+                      entityStart = leadTrustee.entityStart
+                    )
+                  )
+                }
+            }
+
+          updated match {
+            case Success(a) =>
+              Right(Success(a))
+            case Failure(exception) =>
+              Logger.warn(s"[LeadTrusteeIndExtractor] failed to extract data due to ${exception.getMessage}")
+              Left(FailedToExtractData(DisplayTrustLeadTrusteeIndType.toString))
+          }
+      }
+    }
+
+  private def extractNino(leadTrustee: DisplayTrustLeadTrusteeIndType, answers: UserAnswers) = {
+    leadTrustee.identification.nino match {
+      case Some(nino) =>
+        answers.set(TrusteeAUKCitizenPage(0), true)
+          .flatMap(_.set(TrusteesNinoPage(0), nino))
+      case None =>
+        // Assumption that user answered no as nino is not provided
+        answers.set(TrusteeAUKCitizenPage(0), false)
+    }
+  }
+
+  private def extractPassportOrIDCard(leadTrustee: DisplayTrustLeadTrusteeIndType, answers: UserAnswers) = {
+    leadTrustee.identification.passport match {
+      case Some(passport) =>
+        answers.set(TrusteePassportIDCardPage(0), PassportOrIdCardDetails(passport.countryOfIssue, passport.number, passport.expirationDate))
```

Comment:
Should this be setting `TrusteeAUKCitizenPage` as `false`?
Should is also be setting `Do you know passport` to `true`?